src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadExerciseResource.java

```diff
@@ -274,4 +284,51 @@ else if (!authCheckService.isAtLeastTeachingAssistantForExercise(fileUploadExerc
         exerciseService.delete(exerciseId, false, false);
         return ResponseEntity.ok().headers(HeaderUtil.createEntityDeletionAlert(applicationName, true, ENTITY_NAME, fileUploadExercise.getTitle())).build();
     }
+
+    /**
+     * POST /file-upload-exercises/:exerciseId/export-submissions : sends exercise submissions as zip
+     *
+     * @param exerciseId the id of the exercise to get the repos from
+     * @param submissionExportOptions the options that should be used for the export
+     * @return ResponseEntity with status
+     */
+    @PostMapping("/file-upload-exercises/{exerciseId}/export-submissions")
+    @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
+    public ResponseEntity<Resource> exportSubmissions(@PathVariable long exerciseId, @RequestBody SubmissionExportOptionsDTO submissionExportOptions) {
+
+        Optional<FileUploadExercise> optionalFileUploadExercise = fileUploadExerciseRepository.findById(exerciseId);
+        if (optionalFileUploadExercise.isEmpty()) {
+            return notFound();
+        }
+
+        FileUploadExercise fileUploadExercise = optionalFileUploadExercise.get();
+
+        if (!authCheckService.isAtLeastTeachingAssistantForExercise(fileUploadExercise)) {
+            return forbidden();
+        }
+
+        // ta's are not allowed to download all participations
+        if (submissionExportOptions.isExportAllParticipants() && authCheckService.isTeachingAssistantInCourse(fileUploadExercise.getCourseViaExerciseGroupOrCourseMember(), null)) {
```

Comment:
This check caused some problems for me during testing: If you are both instructor/admin AND tutor, you cannot download all submissions.
I do not know if this is a real scenario in normal courses, but in test courses, this can happen.
Maybe you should change the check to something like `submissionExportOptions.isExportAllParticipants() && !authCheckService.isAtLeastInstructorInCourse`.

Also, no error message was shown to me, so I had to check the source code to identify the issue. Maybe you can display error messages in a follow up or so.